_data/events.yml

```diff
@@ -5,7 +5,7 @@
   category: category-book-club
   category_name: Book Club
   date: THU, MAY 30, 2024
-  expiration: "20240531"
+  expiration: "20240530"
```

Comment:
I think the problem here is Jekyll, the dates don't change unless there is a deploy, which is strange. I think we have to do that with javascript instead of jekyl.. The same problem happen with registration of mentors, I had to redeploy the application to "refresh", in that case this change won't fix the problem

Reply:
Oh really? Because I saw that it showed correctly on July 6, but that could have been because of a redeploy as you said.
I'd put this in draft then, let's wait till July 12 to check again (there's an event for 11th).
We can trigger a redeploy then and see if it changes @dricazenck